.gitignore

```diff
@@ -0,0 +1,33 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
+
+# Ignore all logfiles and tempfiles.
+/log/*
+/tmp/*
+!/log/.keep
+!/tmp/.keep
+
+# Ignore pidfiles, but keep the directory.
+/tmp/pids/*
+!/tmp/pids/
+!/tmp/pids/.keep
+
+# Ignore uploaded files in development.
+/storage/*
+!/storage/.keep
+/tmp/storage/*
+!/tmp/storage/
+!/tmp/storage/.keep
+
+# Ignore master key for decrypting credentials and more.
+/config/master.key
```

Comment:
- [ ] You are not sharing to the repo the 'master.key' file.  When I try to create the database, this happens:

`rails db:create`:

![image](https://user-images.githubusercontent.com/57421823/233545862-35bf1530-647a-4026-b040-b7d4681a7889.png)

## Explanation

This means your app requires special procedures to be run properly because your users will need to generate their own `master.key`.

## Tips and extra help

Just include these instructions on your readme:

```
In order to generate your own `master.key` follow these steps:

1. Remove config/master.key and config/credentials.yml.enc if they exist.
2. Run in the terminal: EDITOR=code rails credentials:edit 
3. This command will create a new master.key and credentials.yml.enc if they do not exist.

```
That way, your users will be able to create a database and run the project with their own master key. 💪

Reply:
Thank you @Meltrust , we  will remove it from git ignore and also add the instructions in case they want to create their own 😄